### ISSUE TITLE
`DateTimePatternGeneratorCache::CreateGenerator Allocation failed - process out of memory` when using icu-data-full version: 76.1

### ISSUE THREAD

**Issue by muadnan** [wolfi-dev/os#36852]
I am getting following error when ran node application in a docker image which contains:
icu-data-full version: **76.1**
Fixed after installing: icu-data-full version: **75.1**

**Error:**
```

FATAL ERROR: DateTimePatternGeneratorCache::CreateGenerator Allocation failed - process out of memory |  
-- | --
  |   | ----- Native stack trace ----- |  
  |   |   |  
  |   | 1: 0x62141394dc4d node::OOMErrorHandler(char const*, v8::OOMDetails const&) [node] |  
  |   | 2: 0x621413d3e7b4 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [node] |  
  |   | 3: 0x621413d3ed22 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [node] |  
  |   | 4: 0x6214141c8e05 v8::internal::JSDateTimeFormat::New(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, char const*) [node] |  
  |   | 5: 0x621413ddbf2d v8::internal::Builtin_DateTimeFormatConstructor(int, unsigned long*, v8::internal::Isolate*) [node] |  
  |   | 6: 0x6214148611b6  [node] |  
  |   |   |  
  |   | <--- Last few GCs ---> |  
  |   |   |  
  |   | [7:0x6017079b13a0]      280 ms: Scavenge 23.3 (24.5) -> 22.9 (27.3) MB, 0.52 / 0.00 ms  (average mu = 1.000, current mu = 1.000) allocation failure;
```

**Comment by mkushakov** [comment]
I have also issue with new `icu` and `icu-data-full` packages of `76.1-r0`. When running `npm install` it is blowing with following error:
```
#12 [build 5/9] RUN npm ci
 #12 1.140 npm error Internal error. Icu error.
 #12 1.143 npm error A complete log of this run can be found in: ~/.npm/_logs/2024-12-13T11_21_03_150Z-debug-0.log
```
But working fine if using version `75.1-r3`.

Related to recent upgrade https://github.com/wolfi-dev/os/pull/35274

**Comment by mkushakov** [comment]
OK, issue has been fixed by this PR https://github.com/wolfi-dev/os/pull/36864
thanks

**Comment by muadnan** [comment]
> OK, issue has been fixed by this PR #36864 thanks

thanks for your response closing this issue.
